lastminute.yml

```diff
@@ -193,11 +193,17 @@
     - name: DOMJudge only bits
       when: ccs == "domjudge"
       block:
+        - name: Is contest_detail TeamGuide.pdf available?
+          ansible.builtin.stat:
+            path: files/"{{ contest_detail }}-domjudge-team-manual.pdf"
+          register: domjudge_team_manual
+
         - name: Copy domjudge TeamGuide.pdf (if ccs == domjudge)
           ansible.builtin.copy:
             src: files/"{{ contest_detail }}-domjudge-team-manual.pdf"
             dest: /usr/share/doc/icpc/TeamGuide.pdf
             mode: '0644'
+          when: domjudge_team_manual.stat.exists
```

Comment:
I think you want something like `with_fileglob` https://docs.ansible.com/ansible/latest/collections/ansible/builtin/fileglob_lookup.html, because `stat` will look for that file on the remote system being managed, not your local machine.

Reply:
actually i wanted that checked on the localhost.